galaxy/models.py

```diff
@@ -0,0 +1,378 @@
+    @classmethod
+    def compute_user_score(cls, user):
+        """
+        This compute an individual score for each citizen. It will later be used by the graph algorithm to push
+        higher scores towards the center of the galaxy.
+
+        Idea: This could be added to the computation:
+              - Forum posts
+              - Picture count
+              - Counter consumption
+              - Barman time
+              - ...
+        """
+        user_score = 1
+        user_score += cls.compute_user_family_score(user)
+        user_score += cls.compute_user_pictures_score(user)
+        user_score += cls.compute_user_clubs_score(user)
+
+        # TODO:
+        # Scale that value with some magic number to accommodate to typical data
+        # Really active galaxy citizen after 5 years typically have a score of about XXX
+        # Citizen that were seen regularly without taking much part in organizations typically have a score of about XXX
+        # Citizen that only went to a few events typically score about XXX
+        user_score = int(math.log2(user_score))
+
+        return user_score
+
+    @classmethod
+    def compute_user_family_score(cls, user):
+        user_family_score = (
+            user.godfathers.count() + user.godchildren.count()
+        ) * cls.FAMILY_LINK_POINTS
+        return user_family_score
+
+    @classmethod
+    def compute_user_pictures_score(cls, user):
+        user_pictures_score = user.pictures.count() * cls.PICTURE_POINTS
+        return user_pictures_score
+
+    @classmethod
+    def compute_user_clubs_score(cls, user):
+        user_clubs_score = user.memberships.count() * cls.CLUBS_POINTS
+        return user_clubs_score
+
+    ####################
+    # Inter-user score #
+    ####################
+
+    @classmethod
+    def compute_users_score(cls, user1, user2):
+        family = cls.compute_users_family_score(user1, user2)
+        pictures = cls.compute_users_pictures_score(user1, user2)
+        clubs = cls.compute_users_clubs_score(user1, user2)
+        score = family + pictures + clubs
+        return score, family, pictures, clubs
+
+    @classmethod
+    def compute_users_family_score(cls, user1, user2):
+        link_count = User.objects.filter(
+            Q(id=user1.id, godfathers=user2) | Q(id=user2.id, godfathers=user1)
+        ).count()
+        if link_count > 0:
+            score = link_count * cls.FAMILY_LINK_POINTS
+            print(
+                "    - '%s' and '%s' have %s direct family link"
+                % (user1, user2, link_count)
+            )
+            return score
+        return 0
+
+    @classmethod
+    def compute_users_pictures_score(cls, user1, user2):
+        score = (
+            Picture.objects.filter(people__user__in=(user1,))
+            .filter(people__user__in=(user2,))
+            .count()
+        )
+        print("\t'{user1}' was pictured with '{user2}' {score} times"
```

Comment:
```suggestion
        print("\t'{user1}' was pictured with '{user2}' {score} times")
```

J'ai oublié la parenthèse dans ma suggestion. Désolé.